src/openrct2/core/String.hpp

```diff
@@ -42,6 +42,7 @@ namespace String
     bool IsNullOrEmpty(const utf8* str);
     int32_t Compare(const std::string& a, const std::string& b, bool ignoreCase = false);
     int32_t Compare(const utf8* a, const utf8* b, bool ignoreCase = false);
+    bool Equals(const std::string_view& a, const std::string_view& b, bool ignoreCase);
```

Comment:
Should this have an `ignoreCase` default equal to false? (all other similar alternatives do...

Reply:
Yes, that was done in a later commit